openedx/core/djangoapps/oauth_dispatch/management/commands/generate_jwt_signing_key.py

```diff
@@ -0,0 +1,117 @@
+"""
+Management command for generating an asymmetric keypair to sign JSON Web Tokens.
+"""
+
+from __future__ import unicode_literals
+
+import logging
+import json
+import random
+import string
+
+from django.conf import settings
+from django.core.management.base import BaseCommand
+
+from Cryptodome.PublicKey import RSA
+from jwkest import jwk
+
+
+log = logging.getLogger(__name__)
+
+
+class Command(BaseCommand):
+    """
+    Generates an asymmetric keypair to sign JSON Web Tokens.
+    """
```

Comment:
Might be good to add some details here about when/why you would use this command along with an example command line.

Reply:
Good idea.  Done.  Take a look at the new [help](https://github.com/edx/edx-platform/pull/18681/files#diff-ac2131948b7a7b1fa2eb373f77c64ffeR26).